MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -93,7 +93,8 @@ open class BaseInstructionsBannerView: UIControl {
     /**
      Updates the instructions banner info with a given `VisualInstructionBanner`.
      */
-    public func updateInstruction(_ instruction: VisualInstructionBanner?) {
+    @objc(updateForVisualInstruction:)
```

Comment:
This should be `-updateForVisualInstructionBanner:`, given the parameter’s type.